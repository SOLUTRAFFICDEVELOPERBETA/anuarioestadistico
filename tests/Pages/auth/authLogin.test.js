import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import AuthLogin from '../../../pages/auth';

describe('Pruebas en <AuthLogin/>', ()=> {

    const wrapper = shallow(
        <AuthLogin
        />
    )

    test('debe mostrarse correctamente', () => {
        expect(wrapper).toMatchSnapshot();
    })

    test('debe mostrar el componente <LogInForm/>', ()=> {
        expect(wrapper.find('LogInForm').exists()).toBe(true);
    })
})