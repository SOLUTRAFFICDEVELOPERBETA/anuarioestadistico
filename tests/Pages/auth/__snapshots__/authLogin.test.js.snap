// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <AuthLogin/> debe mostrarse correctamente 1`] = `
<AuthLayout>
  <LogInForm />
</AuthLayout>
`;
